setup.py

```diff
@@ -109,7 +109,7 @@
         'Nano-Utils>=0.4.3',
         'numpy',
         'scipy',
-        'pandas<1.0.0',
+        'pandas>=1.0.0',
```

Comment:
```suggestion
        'pandas',
```

Is there a reason why `'pandas>=1.0.0'` and not just plain `'pandas'`?

Reply:
I agree with plain pandas and let pip to figure out which one :)